DBison.WPF/MainWindowViewModel.cs

```diff
@@ -461,6 +495,19 @@ private void __SettingsHandler_SettingChanged(object? sender, Core.EventArgument
         {
             m_ExecutionTimer.Interval = TimeSpan.FromSeconds(Settings.FilterUpdateRate);
         }
+        else if (e.ChangedSettingName == nameof(Settings.PluginPath))
```

Comment:
We have now that many that we should use switch instead of if else if else if else if .... else